components/automate-cli/cmd/chef-automate/constants.go

```diff
@@ -1,13 +1,23 @@
 package main
 
 const (
-	FRONTEND      = "frontend"
-	AUTOMATE      = "automate"
-	CHEF_SERVER   = "chef_server"
-	POSTGRESQL    = "postgresql"
-	OPENSEARCH    = "opensearch"
-	SET           = "set"
-	PATCH         = "patch"
-	SUDO_PASSWORD = "sudo_password"
-	IPV4REGEX     = `^(((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)(\.|$)){4})`
+	FRONTEND            = "frontend"
+	AUTOMATE            = "automate"
+	CHEF_SERVER         = "chef_server"
+	POSTGRESQL          = "postgresql"
+	OPENSEARCH          = "opensearch"
+	SET                 = "set"
+	PATCH               = "patch"
+	SUDO_PASSWORD       = "sudo_password"
+	IPV4REGEX           = `^(((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)(\.|$)){4})`
+	A2VERSIONCMD        = "sudo chef-automate version"
+	A2VERSIONVERBOSE    = "sudo chef-automate version -v "
+	CSVERSIONCMD        = "sudo chef-server-ctl version"
+	OSGETINFOCURLCMD    = "curl -XGET http://localhost:10144"
+	PGGETVERSIONCURLCMD = "PGPASSWORD=%s  hab pkg exec core/postgresql13  psql -U %s -h localhost -p 10145 -d postgres --dbname postgres -tAc 'SELECT version()'"
```

Comment:
Can we have a version constant, what if the version gets changed in future, then this command will not work.